apps/server/src/trpc/graph.ts

```diff
@@ -145,9 +166,12 @@ export const graph = createRouter()
       ),
     }),
     async resolve({ input }) {
-      return api.graphRepo.update(input.graph).then((res) => ({
-        ...res,
-        graph: flatten.graph(res.graph),
-      }))
+      return api.graphRepo
+        .findOneById(input.graphId)
+        .then((g) => api.graphRepo.update(g, input.graph))
+        .then((res) => ({
+          ...res,
+          graph: flatten.graph(res.graph),
+        }))
     },
```

Comment:
For route standardization like `/entity/entityId/method`, `graphId` is now a parameter for `graph/{graphId}/update`.

I chose to refactor this way because it follows our convention so far, where we call `findOneById` in the `resolve`, then pass the entity to the repo function